azure-pipelines.yml

```diff
@@ -64,10 +64,10 @@ jobs:
       sudo apt autoremove
       sudo apt install --reinstall -y gcc-9 g++-9 libstdc++-9-dev
       echo ========================= install Intel OneAPI ===============================
-      sudo apt-get install intel-oneapi-mkl-2021.1-beta10                \
-                           intel-oneapi-mkl-devel-2021.1-beta10          \
-                           intel-oneapi-dpcpp-cpp-compiler-2021.1-beta10 \
-                           intel-oneapi-dpcpp-cpp-2021.1-beta10
+      sudo apt-get install intel-oneapi-mkl                \
+                           intel-oneapi-mkl-devel          \
+                           intel-oneapi-dpcpp-cpp-compiler \
+                           intel-oneapi-dpcpp-cpp
```

Comment:
I'm not sure the package `intel-oneapi-dpcpp-cpp` exists.

Reply:
It looks like they renamed something again.

Reply:
```
The following additional packages will be installed:
  intel-oneapi-common-licensing intel-oneapi-common-licensing-2021.1.1
  intel-oneapi-common-vars intel-oneapi-compiler-cpp-eclipse-cfg
  intel-oneapi-compiler-dpcpp-cpp-2021.1.1
  intel-oneapi-compiler-dpcpp-cpp-common-2021.1.1
  intel-oneapi-compiler-dpcpp-cpp-runtime
  intel-oneapi-compiler-dpcpp-cpp-runtime-2021.1.1

```